app/views/resources/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "Ressources externes") %>
+<% provide(:active_section, "speedcubing") %>
+<%= render "layouts/page_wrapper", title:"Ressources externes" do %>
+  <% @resources.each do |resource| %>
+    <h5>
+      <a href="<%= resource[1]['link'] %>" target=_blank><%= resource[1]['name'] %></a>
+    </h5>
+    <img src="<%= resource[1]['img'] %>" alt="Logo <%= resource[1]['name'] %>">
+    <p><%= resource[1]['description'] %></p>
```

Comment:
Ici tu te retrouves à faire `resource[1]` à chaque fois, ce qui n'est pas top.
`@resources` est une map (enfin un `Hash`), donc ton itérateur [`each`](https://docs.ruby-lang.org/en/2.0.0/Hash.html#method-i-each) peut te passer à la fois la clé et la valeur de l'entrée courante de la map :
```ruby
irb(main):008:0> resources.each { |key, val| puts "#{key} : #{val.keys}" }
wca : ["name", "link", "description", "img"]
francocube : ["name", "description", "link", "img"]
```

Ici tu pourrais donc faire directement `val['description']`.